packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -226,7 +231,7 @@ export function SchedulesTable({
         {!minimal && (
           <Field width={80} style={{ textAlign: 'center' }}>
             {item._date && item._date.frequency && (
-              <Check style={{ width: 13, height: 13 }} />
+              <Check style={{ width: 13, height: 13, color: 'inherit' }} />
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this necessary?

Reply:
Removed